eng/versioning/external_dependencies.txt

```diff
@@ -7,6 +7,7 @@
 
 # Format;
 # groupId:artifactId;dependency-version
+com.azure:azure-communication-common;1.0.0
```

Comment:
In this particular change, we ARE treating common 1.0.0 as 3rd party since we want PhoneNumber 1.0.1 to reference this version instead of the current version of Common which is not released.

Reply:
I guess I misunderstood the versioning mechanism. I'll give this a try.

Reply:
Yes, sorry, I meant `version_client.txt`